### PR TITLE
First draft of technical documentation of the Widget

### PR DESCRIPTION
**Pull request by robinandeer**
## Describe your changes

See title.

## Checklist before requesting a review

- [ ] I have performed a self-review of my code

### CODE REVIEW

**Comment by robinandeer** [comment]
Current dependencies on/for this PR:
* main
  * **PR #3583** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3583?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
    * **PR #3584** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3584?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
      * **PR #3586** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3586?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3586?utm_source=stack-comment).

**Review by alebedev — APPROVED** [review]
Very nice to have the docs where the code is!

File: apps/store/src/features/widget/README.md [hunk 1]
```diff
@@ -1,5 +1,57 @@
...
+
+## Architecture
+
+The widget is built around the same concepts as the rest of the site. However, it doesn't use either of the Shop Session or Price Intent contexts. Instead it refers to these enteties directly in the URL.
+
+The widget also makes use of SSR to render the pages in the flow.
+
+## Landing Page (optional)
+
+Any CMS page can act as a landing page for the widget. The only requirement is to includ a link to a _Widget Flow_. Multiple landing pages can link to the same _Widget Flow_.
```
Comment (guilhermespopolin) [inline apps/store/src/features/widget/README.md:13]:
```suggestion
Any CMS page can act as a landing page for the widget. The only requirement is to include a link to a _Widget Flow_. Multiple landing pages can link to the same _Widget Flow_.
```

**Review by guilhermespopolin — APPROVED** [review]

**Comment by robinandeer** [comment]
### Merge activity

* **Nov 20, 9:38 AM**: [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3586) rebased this pull request as part of a merge.

**Pull request merged**